app/controllers/buyers/accounts_controller.rb

```diff
@@ -52,6 +52,7 @@ def create
       redirect_to admin_buyers_account_path(@buyer)
     else
       @user = signup_result.user
+      flash[:error] = signup_result.errors.messages.without(:account, :user).values.join(', ')
```

Comment:
`full_messages.to_sentence` cannot be used?

Reply:
It is possible but redundant because the errors of `account` and `user` already appear right below the attribute that has the error.